Userland/Games/Chess/ChessWidget.h

```diff
@@ -43,7 +43,7 @@ class ChessWidget final
     void set_side(Chess::Color side) { m_side = side; };
 
     void set_piece_set(StringView set);
-    DeprecatedString const& piece_set() const { return m_piece_set; };
+    StringView const& piece_set() const { return m_piece_set; };
```

Comment:
It looks like this function isn't used, so we should probably get rid of it instead.
And note that providing an API to access data that we don't own is probably a bit silly.